lib/timers.js

```diff
@@ -502,24 +502,26 @@ Timeout.prototype.close = function() {
 };
 
 
-var immediateQueue = {};
-L.init(immediateQueue);
+var immediateQueue = L.create();
```

Comment:
@andrasq conveniently, [here is a StackOverflow answer I wrote](http://stackoverflow.com/questions/24987896/how-does-bluebirds-util-tofastproperties-function-make-an-objects-properties) about a technique petkaantonov used in bluebird.